### PR TITLE
matek hc04 troubleshooting

### PR DESCRIPTION
**Pull request by jlpoltrack**
Title

### CODE REVIEW

**Comment by jlpoltrack** [comment]
@olliw42 Could you kindly review?

File: docs/MATEKSYS.md [hunk 1]
```diff
@@ -28,3 +28,12 @@ Links to the MatekSys website are found below, which include product specificati
 The most convenient way of flashing the mLRS devices is via their USB-C port and using the [mLRS Web Flasher](https://mlrs.xyz/flash) app.
 
 Plug in the USB-C cable while pressing the bind button. This puts the mLRS device into DFU mode (LEDs are not flashing). Then follow the instructions provided by the mLRS Web Flasher app.
+
+## Tx Module | HC-04 Bluetooth Troubleshooting ##
```
Comment (olliw42) [inline docs/MATEKSYS.md:32]: maybe the Troubleshooting can go, as it also is kind of a normal how to use

File: docs/MATEKSYS.md [hunk 2]
```diff
@@ -28,3 +28,12 @@ Links to the MatekSys website are found below, which include product specificati
...
+
+To use the HC-04 Bluetooth module on Matek mLRS Tx modules, no additional configuration should be needed as mLRS will automatically configure all of the necessary settings on the module. However, if you're having problems you can check the following items:
```
Comment (olliw42) [inline docs/MATEKSYS.md:34]: maybe more precise, "... on MatekSys mR24-30/mR900-30 Tx module kits, no additional configuration of the HC04 module ... on the HC04 module".

File: docs/MATEKSYS.md [hunk 3]
```diff
@@ -28,3 +28,12 @@ Links to the MatekSys website are found below, which include product specificati
...
+
+- Ensure that the 3 dip switches are all in the 'ON' / left position when looking at the Tx module with the antenna pointing up. PShoto:
```
Comment (olliw42) [inline docs/MATEKSYS.md:36]: what is this PShoto for?

**Comment by olliw42** [comment]
sorry, missed that somewhat.

MANY THX, I guess this will be much appreciated

File: docs/MATEKSYS.md (hunk 1 above)
Comment (jlpoltrack) [reply docs/MATEKSYS.md:32]: Title updated

File: docs/MATEKSYS.md (hunk 2 above)
Comment (jlpoltrack) [reply docs/MATEKSYS.md:34]: Added

File: docs/MATEKSYS.md (hunk 3 above)
Comment (jlpoltrack) [reply docs/MATEKSYS.md:36]: Well it was was supposed to be 'Photo' but just removed.

**Pull request merged**